### DISCUSSION TITLE
TypeError: Cannot use 'in' operator to search for '../node_modules/webpack/hot/dev-server.js' in undefined

### DISCUSSION THREAD

**Discussion by nextniko** [webpack/webpack#17900] [Q&A]
# Bug report

**What is the current behavior?**
I encountered an error when using DllReferencePlugin
The error message is TypeError: Cannot use 'in' operator to search for '../node_modules/webpack/hot/dev-server.js' in undefined

**If the current behavior is a bug, please provide the steps to reproduce.**
My Code：
new webpack.DllPlugin({ context: __dirname, name: '[name]_[fullhash]', path: path.join(__dirname, './dll/manifest.json'), }), new webpack.DllReferencePlugin({ context: __dirname, manifest: path.join(__dirname, './dll/manifest.json'), sourceType: 'commonjs2' })

Error location：
lib/DelegatedModuleFactoryPlugin.js
![291453612-9304888f-1671-4ba6-944b-fb735dfc8c0e](https://github.com/webpack/webpack/assets/40553790/eaf52f3e-dcf2-4ed5-a3a0-e39449e718a6)
**What is the expected behavior?**
Resolve errors
**Other relevant information:**
webpack version: 3.6.0
Node.js version: 14.18.0
Operating System: windows
Additional tools:

**Comment by alexander-akait** [comment]
Please create reproducible test repo

**Comment by nextniko** [comment]
I'm sorry, I don't know how to create reproducible test repo

**Comment by alexander-akait** [comment]
How I can reproduce this problem? Can you provide full configuration and small example of a code to reproduce your problem, thank you

**Comment by alexander-akait** [comment]
Do you real use `webpack version: 3.6.0`?
